package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "test": "standard && hallmark && nyc node test.js",
     "test-browsers-local": "airtap --coverage test.js",
-    "coverage": "nyc report --reporter=text-lcov | coveralls",
```

Comment:
Can you run `npm rm coveralls --save-dev`? Thank you!

Reply:
Removed in 51bf970